ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6620,3 +6620,27 @@ VMADC:
   - mixed_air_damper_percentage_sensor
   implements:
   - CONTROL
+
+
+SEWTM:
+  description: "Secondary water temperature monitoring"
+  opt_uses:
+  - secondary_supply_water_temperature_sensor
+  uses:
+  - secondary_return_water_temperature_sensor
+  implements:
+  - MONITORING
+
+HWTTCX2:
+  description: "Hot water tank temperature control X2 with single setpoint"
+  is_abstract: true
+  opt_uses: 
+  - heater_run_command_1
```

Comment:
Hot water tanks typically do not have heating mechanisms, please explain the use of the `heater_run_command` fields or remove them from this type.